### ISSUE TITLE
Refiner.get_current_mesh() raises TypeError when called

### ISSUE THREAD

**Issue by mattwala** [inducer/meshmode#4]
Eg.

```
>>> r = Refiner(m)
>>> r.refine(np.ones(m.nelements))
32
>>> r.get_current_mesh()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/matt/src/meshmode/meshmode/mesh/refinement/__init__.py", line 239, in get_current_mesh
    len(self.last_mesh.vertices[0])),
TypeError: generate_nodal_adjacency() missing 1 required positional argument: 'groups'
```

**Comment by inducer** [comment]
@guptashvm, could you take a look?

**Comment by mattwala** [comment]
This was fixed by e0d2682e7d555d7783c977fde609803750516f68.
